src/filldist.jl

```diff
@@ -11,7 +11,7 @@ const FillVectorOfUnivariate{
 function filldist(dist::UnivariateDistribution, N::Int)
     return product_distribution(Fill(dist, N))
 end
-filldist(d::Normal, N::Int) = MvNormal(fill(d.μ, N), d.σ)
+filldist(d::Normal, N::Int) = TuringMvNormal(fill(d.μ, N), d.σ)
```

Comment:
Maybe use `Fill` instead of `fill`?

Reply:
`Fill` doesn't work well with Tracker. `fill` gives a `TrackedArray`.

Reply:
Ah OK. But isn't the point here that we don't need an actual array?

Reply:
TrackedArrays are better for broadcasting. We have broadcasting in MvNormal logpdf.

Reply:
I see, thanks for explaining!